app/admin/assistance.rb

```diff
@@ -4,7 +4,15 @@
   menu parent: :questions, priority: 2
 
   permit_params do
-    permitted = %i[question_id expert_id institution_id title email_specific_sentence]
+    permitted = [
+      :id,
+      :_destroy,
```

Comment:
To move downwards. Try to order fields according to their importance (id -> title -> question_id -> institution_id > email_... -> _destroy)